src/test/resources/log4j2.properties

```diff
@@ -0,0 +1,23 @@
+rootLogger.level = fatal
+rootLogger.appenderRef.stdout.ref = STDOUT
+logger.app.name=com.cse.ngsa
+logger.app.level=fatal
+logger.netty.name = io.netty
+logger.netty.level = fatal
+logger.reactor.name = io.reactivex
+logger.reactor.level = fatal
+# New Logger for RequestLogger object only
+logger.request.name=com.cse.ngsa.app.middleware.RequestLogger
+logger.request.level=fatal
+logger.request.appenderRef.stdout.ref=STDOUT_Request
+logger.request.additivity=false
+# New STDOUT Appender for Requestlogger
+appender.console_req.name = STDOUT_Request
+appender.console_req.type = Console
+appender.console_req.layout.type = PatternLayout
+appender.console_req.layout.pattern = %msg%n
+# STDOUT is a ConsoleAppender and uses PatternLayout.
+appender.console.name = STDOUT
+appender.console.type = Console
+appender.console.layout.type = PatternLayout
+appender.console.layout.pattern = %d{yyyy-MM-dd HH:mm:ss.SSS} [%thread] %-5level %logger{36}.%M \\(%line\\) - %msg%n
```

Comment:
Add a new line to the end of the file.

Do we need the previous tab-delimited stdout logger?  I think the ngsa-app C# only uses JSON for log output.

Reply:
@jofultz The last 4 lines are just creating a new appender named STDOUT. I believe log4j2 also have a default appender of that name. Tests run fine without the appender (last 4 lines). I'll delete the appender.